src/libs/ajax/Groups.ts

```diff
@@ -53,8 +54,28 @@ export const Groups = (signal?: AbortSignal) => ({
       },
 
       addUsers: (roles: GroupRole[], emails: string[]): Promise<void[]> => {
```

Comment:
I did not notice this before, but why is `roles` and array? Adding all emails to all roles does not seem right.

Reply:
I believe it was initially designed to handle multiple role assignments to a single email but It only submits a single role in this case: https://github.com/DataBiosphere/terra-ui/blob/b520887ac1a941ab35f164bc4e95ced657548c6f/src/groups/Members/NewMemberModal.tsx#L61

Reply:
I think you should change it to be singular. I don't think there is a use case for multiple roles since groups have only 2 roles and adding to both at the same time makes no sense.

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5227/commits/7011c873cde9e1bbf7732b2a4e059d923a8c0a49